README.md

```diff
@@ -1,68 +1,212 @@
+| CLOUDAGENT_ADMIN_WS_ORIGIN     | y        | ws://localhost:3100                                            | veritable-cloudagent web socker address/url                                                      |
+| INVITATION_PIN_SECRET          | y        | Buffer.from('secret', 'utf8')                                  | an encoded array of utf-8 format                                                                 |
+| INVITATION_PIN_ATTEMPT_LIMIT   | y        | 5                                                              | number of allowed pin validation attempts                                                        |
+| INVITATION_FROM_COMPANY_NUMBER | n        | 07964699                                                       | default company number, currently is us                                                          |
+| ISSUANCE_DID_POLICY            | y        | EXISTING_OR_NEW                                                | DID and either create or use existing: [CREATE_NEW, FIND_EXIStING, EXISTING_OR_NEW, did:somedid] |
+| ISSUANCE_SCHEMA_POLICY         | y        | EXISTING_OR_NEW                                                | Same as above but for credential schema                                                          |
+| ISSUANCE_CRED_DEF_POLICY       | y        | EXISTING_OR_NEW                                                | Same as above but this is for credential definitions                                             |
+
+## Testing
+Currently this repository consist of two test types: [**integration**, **unit**] and we are using a combination of `mocha`, `chai` and `sinon` frameworks
+
+### Unit Testing
+Unit tests in this repository are done per service/module or class, and follow the bellow pattern.
+```sh
+│   ├── __tests__
+│   │   ├── auth.test.ts
+│   │   ├── __tests__
+│   │   │   ├── index.test.ts
+│   │   │   └── newConnection.test.ts
+│       ├── __tests__
+│       │   └── index.test.ts
+│   ├── __tests__
+│   │   ├── companyHouseEntity.test.ts
+│   │   ├── credentialSchema.test.ts
+│   │   ├── idpService.test.ts
+│   │   └── veritableCloudagent.test.ts
+│   │   ├── __tests__
+│   │   │   └── index.test.ts
+│   │   ├── __tests__
+│   │   │   ├── companyDetailsV1.test.ts
+│   │   │   └── index.test.ts
+│   ├── __tests__
+│   │   │   └── testIndexedAsyncEventEmitter.ts
+│   │   ├── indexedAsyncEventEmitter.test.ts
+│   │   └── twoWayMap.test.ts
+    │   ├── __tests__
+    │   │   ├── connection.test.ts
+    │   │   └── connection.test.ts.snap
+    │   ├── __tests__
+    │   ├── __tests__
+    │   │   ├── fromInvite.test.ts
+    │   │   ├── fromInvite.test.ts.snap
+    │   │   ├── newInvite.test.ts
+    │   │   ├── newInvite.test.ts.snap
+    │   │   ├── pinSubmission.test.ts
+    │   │   └── pinSubmission.test.ts.snap
+    │   ├── __tests__
+    │   │   ├── queries.test.ts
+    │   │   ├── queries.test.ts.snap
+    │   │   ├── queriesList.test.ts
+    │   │   └── queriesList.test.ts.snap
+``` 
```

Comment:
This will be a hard to maintain list. Instead just say unit tests are under `__tests__` directories colocated with the units they test